src/main/java/core/basesyntax/DiscountStrategy.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+import core.basesyntax.impl.BirthdayDiscountService;
+import core.basesyntax.impl.BlackFridayDiscountService;
+import core.basesyntax.impl.DefaultDiscountService;
+import core.basesyntax.impl.NewYearDiscountService;
+import core.basesyntax.strategy.DiscountService;
+
+public class DiscountStrategy {
+    public DiscountService getDiscountServiceBySpecialEvent(String specialEvent) {
+        BlackFridayDiscountService blackFridayDiscountService = new BlackFridayDiscountService();
+        BirthdayDiscountService birthdayDiscountService = new BirthdayDiscountService();
+        NewYearDiscountService newYearDiscountService = new NewYearDiscountService();
+        DefaultDiscountService defaultDiscountService = new DefaultDiscountService();
```

Comment:
you could inline these in returns

Reply:
I know It, but if we want to use each discount several times, we will create new object every time